docs/excel/excel-add-ins-undo-capabilities.md

```diff
@@ -0,0 +1,91 @@
+
+```js
+    await Excel.run({ mergeUndoGroup: true }, async (context) => { 
+        const sheet = context.workbook.worksheets.getActiveWorksheet(); 
+        let range = sheet.getRange("A1"); 
+        range.values = [["123"]]; 
+        
+        await context.sync(); 
+        
+        range = sheet.getRange("B2"); 
+        range.values = [["456"]];
+ 
+        await context.sync(); 
+    }); 
+```
+
+## Unsupported APIs
+
+Most Excel JavaScript APIs support undo actions. See the following table for a list of APIs that don't support undo behavior.
+
+> [!TIP]
+> If you call an unsupported API in your add-in, the user’s undo stack is cleared starting from that API call, and a user cannot undo actions past that point.
+
+| API | Supported in Excel on the web | Supported in Excel on Windows and Excel on Mac | Notes |
+|:----------|:--------------|:------|
+| `AllowEditRange.address` | No | No | *None* |
+| `AllowEditRange.delete` | No | No | *None* |
+| `AllowEditRange.pauseProtection` | No | No | *None* |
+| `AllowEditRange.setPassword` | No | No | *None* |
+| `AllowEditRange.title` | No | No | *None* |
+| `AllowEditRangeCollection.add` | No | No | *None* |
+| `AllowEditRangeCollection.pauseProtection` | No | No | *None* |
+| `Chart.categoryLabelLevel` | No | No | *None* |
+| `Chart.seriesNameLevel` | No | No | *None* |
+| `DataConnectionCollection.refreshAll` | No | No | *None* |
+| `LinkedWorkbook.refresh` | No | No | *None* |
+| `LinkedWorkbookCollection.refreshAll` | No | No | *None* |
+| `Workbook.close` | No | No | *None* |
+| `Workbook.insertWorksheetsFromBase64` | No | No | *None* |
+| `Workbook.save` | No | No | *None* |
+| `WorkbookProtection.protect` | No | No | *None* |
+| `WorkbookProtection.unprotect` | No | No | *None* |
+| `Worksheet.copy` | No | No | *None* |
+| `Worksheet.delete` | No | No | *None* |
+| `Worksheet.name` | Yes | No | *None* |
+| `Worksheet.position` | Yes | No | *None* |
+| `Worksheet.visibility​` | Yes | No | *None* |
+| `WorksheetCollection.addFromBase64` | No | No | *None* |
+| `WorksheetProtection.pauseProtection` | No | No | *None* |
+| `WorksheetProtection.protect` | No | No | *None* |
+| `WorksheetProtection.resumeProtection` | No | No | *None* |
+| `WorksheetProtection.setPassword` | No | No | *None* |
+| `WorksheetProtection.unprotect` | No | No | *None* |
+| `WorksheetProtection.updateOptions` | No | No | *None* |
+
+## Check for undo support
+
+Use the`isSetSupported` method of the [OfficeRuntime.ApiInformation](/javascript/api/office-runtime/officeruntime.apiinformation) interface to check for undo support and provide a fallback experience if undo support isn't available. See the following code sample for more information about how to use `isSetSupported`.
+
+```js
+    if (Office.context.requirements.isSetSupported("ExcelUndoApiAll", "1.0")) { 
```

Comment:
I'm surprised this is "1.0" instead of "1.1", which seems to be the standard starting point in our requirement sets (not that starting with the logical "1.0" is bad).

Reply:
Good question! I'll check with the PM.